tests/e2e/TimeDiff.js

```diff
@@ -0,0 +1,30 @@
+import 'testcafe';
+import LayoutPage from './page-objects/Layout';
+import HomePage from './page-objects/Homepage';
+import RelatedModels from './page-objects/RelatedModels';
+import TimeDiff from './page-objects/TimeDiff';
+
+const layoutPage = new LayoutPage();
+const homePage = new HomePage();
+const relatedModels = new RelatedModels();
+const timeDiff = new TimeDiff();
+
+fixture('Measuring execution time of liquid code fragments').page(
+  layoutPage.URL.staging
+);
+
+test('There are no liquid errors on the page', async t => {
+  await t.click(homePage.link.timeDiff);
+  await layoutPage.checkLiquidErrors();
+  await t.expect(timeDiff.link.documentation.exists).ok();
```

Comment:
this line changes the purpose of this test - either change title or move it somewhere else (there are before/after hooks if thats something you need)

Reply:
Fixed